packages/pieces/src/lib/apps/drip/actions/add-subscriber-to-campaign.action.ts

```diff
@@ -0,0 +1,74 @@
+import { httpClient } from "../../../common/http/core/http-client";
+import { HttpMethod } from "../../../common/http/core/http-method";
+import { HttpRequest } from "../../../common/http/core/http-request";
+import { createAction } from "../../../framework/action/action";
+import { Property } from "../../../framework/property";
+import { dripCommon } from "../common";
+
+export const dripAddSubscriberToCampaign = createAction({
+    name: 'add_subscriber_to_campaign',
+    description: 'Add a subscriber to a campaign (Email series)',
+    displayName: 'Add a subscriber to a campaign',
+    props: {
+        authentication: dripCommon.authentication,
+        account_id: dripCommon.account_id,
+        campaign_id: Property.Dropdown({
```

Comment:
Change the Dropdown in the frontend, to show no options when there is zero options, currently the dropdown doesn't open.

Reply:
fixed